src/components/Publish/Pricing/Fees.tsx

```diff
@@ -43,13 +44,17 @@ export default function Fees({
   pricingType: 'dynamic' | 'fixed'
 }): ReactElement {
   const [field, meta] = useField('pricing.swapFee')
-  const [opcFees, setOpcFees] = useState<OpcFeesData>(undefined)
+  const [communityFee, setCommunityFee] = useState<string>('')
```

Comment:
good thinking making variable consistent to what we display as this stays confusing in code (like, `response.swapOceanFee` is the "community fee" coming from `opcFees` query). But this raises the question what exact "community fee" this actually is and f we should update the UI label too.

Based on [our docs](https://docs.oceanprotocol.com/concepts/fees/#ocean-community-fee), there is a "Ocean Community swap fee" and a "Ocean Community order fee". If we use those 2 distinctions in the docs then we need to have this distinction in our UI too

Reply:
yep , we need both swap and consume (this is how they are called in the sub query)

Reply:
aaaand i see a redesign incoming 😆